samples/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MongoDB.Driver" Version="2.12.3" />
```

Comment:
Not needed.

Reply:
This is needed for mongodb sample query. @pellared asked to integrate the sample into asp.net sample.

Reply:
I saw references to MongoDb on samples/Samples.AspNetCoreMvc21/Samples.AspNetCoreMvc21.csproj this project here doesn't have any.

Reply:
The controllers from mvc21 project are linked to mvc31 project... so the mvc31 technically doesn't contain any logic.